E1-ALRF/ALLP/README.md

```diff
@@ -0,0 +1,87 @@
+## Summary
+
+*Include a short description of the test case.*
+
+*Optional: if your data comes from a specific Storyline or unit test of the IFC Infrastructure Deployment Project or IFC Rail Project, please include a link to the originating documentation here. so additional background information can be created.*
+
+With these instructions the baseline stationing and linear positioning exchange will be established between various domains. This includes basic alignment featuring the common layout described in TC1.1:
+- horizontal
+- vertical
+
+and the stationing and relative positioning concepts where physical elements can be placed relative to positioning elements. In order to accomplish a semantic description of a station, property sets will be used on referents.
```

Comment:
You didn't list Property sets (neither entities nor concepts) in the itemized roots section.

Reply:
Where do you see a placeholder for that? I did not see it.

Reply:
I meant `IfcPropertySet` and `IfcSingleProperty` (or similar) in the *Itemized Roots* section of the document.

Reply:
Not really sure honestly. Were these supposed to be added? 😄

Reply:
It says above:

> property sets will be used on referents

That is the reason for my question. I'm not sure if these are supposed to be added.

Reply:
In "my" TIs, I listed e.g. IfcPropertySet, IfcPropertySingleValue under "Itemised roots/IFC Entities" and e.g. Property Sets For Objects under "Itemised roots/Concept templates".